assets/src/scripts/web-services/observations.js

```diff
@@ -34,4 +34,14 @@ export const fetchAvailableDVTimeSeries = function(monitoringLocationId) {
  */
 export const fetchDVTimeSeries = function(monitoringLocationId, timeSeriesId) {
     return fetchObservationsData(`items/${monitoringLocationId}/observations/statistical-time-series/${timeSeriesId}`);
+};
+
+/*
+ * Fetches the observation data with siteid and returns a Promise
+ * that returns a GeoJson object containing the observation geojson
+ * @param {String} monitoringLocation
+ * @return {Promise}<Object>
+ */
+export const fetchObservationItem = function(monitoringLocationId) {
```

Comment:
Can we call this something more specific list fetch MonitoringLocationMetaData? fetchObservationData doesn't mean much to a future reader of this code.

Reply:
changed fetchObservationItem to fetchMonitoringLocationMetaData.